Makefile

```diff
@@ -11,6 +11,7 @@ endif
 
 VERSION = $(shell git describe --dirty --tags --always)
 GIT_COMMIT = $(shell git rev-parse HEAD)
+K8S_VERSION = v1.16.3
 REPO = github.com/operator-framework/operator-sdk
 BUILD_PATH = $(REPO)/cmd/operator-sdk
```

Comment:
Hi @joelanford @theishshah, 

I did not get this one. 
How a make target will attend the users who install the project from brew or binary and are not using the master? 
Also, what means the k8s version info that will output via the `operator-sdk version` command?
How it will works?

Reply:
This value gets embedded into the binary during build time, so when we run `make install` or run the release script, the resulting binaries will have this value included.

Here's a blog post about it if you're interested in learning more: https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications

Reply:
Really thank you for share.